watertap/unit_models/reverse_osmosis_0D.py

```diff
@@ -205,46 +86,112 @@ def build(self):
             balance_type=self.config.momentum_balance_type,
             has_pressure_change=self.config.has_pressure_change)
 
-        # Build control volume for permeate side
-        self.permeate_side = ControlVolume0DBlock(default={
-            "dynamic": False,
-            "has_holdup": False,
-            "property_package": self.config.property_package,
-            "property_package_args": self.config.property_package_args})
-
-        self.permeate_side.add_state_blocks(
-            has_phase_equilibrium=False)
+        # for quacking like 1D model
+        add_object_reference(self.feed_side,'properties',
+                {**{(t,0.) : self.feed_side.properties_in[t] for t in self.flowsheet().config.time},
+                 **{(t,1.) : self.feed_side.properties_out[t] for t in self.flowsheet().config.time}}
+                )
```

Comment:
Nice. I was wondering how we could centralize things across the 0D and 1D stateblocks.